trunk/SUAVE/Methods/Missions/Segments/Transition/Lift_Cruise_Optimized.py

```diff
@@ -0,0 +1,250 @@
+    conditions.propulsion.battery_voltage_under_load       = Voltage[:,0] 
+    
+    conditions.frames.inertial.position_vector[:,2]        = -alt # z points down
+    conditions.freestream.altitude[:,0]                    =  alt # positive altitude in this context      
+
+## @ingroup Methods-Missions-Segments-Climb   
+def update_differentials(segment):
+    """ On each iteration creates the differentials and integration funcitons from knowns about the problem. Sets the time at each point. Must return in dimensional time, with t[0] = 0. This is different from the common method as it also includes the scaling of operators.
+
+        Assumptions:
+        Works with a segment discretized in vertical position, altitude
+
+        Inputs:
+        state.numerics.dimensionless.control_points      [Unitless]
+        state.numerics.dimensionless.differentiate       [Unitless]
+        state.numerics.dimensionless.integrate           [Unitless]
+        state.conditions.frames.inertial.position_vector [meter]
+        state.conditions.frames.inertial.velocity_vector [meter/second]
+        
+
+        Outputs:
+        state.conditions.frames.inertial.time            [second]
+
+    """    
+
+    # unpack
+    numerics   = segment.state.numerics
+    conditions = segment.state.conditions
+    initials   = segment.state.initials
+    x          = numerics.dimensionless.control_points
+    D          = numerics.dimensionless.differentiate
+    I          = numerics.dimensionless.integrate    
+    v0         = segment.air_speed_start
+    vf         = segment.air_speed_end  
+    ax         = segment.acceleration   
+    T0         = segment.pitch_initial
+    Tf         = segment.pitch_final     
+    
+    dv = vf - v0
+    acc = -ax  # maintain column array
+
+    # get overall time step
+    dt = -(dv/acc)
+
+    # rescale operators
+    x = x * dt
+    D = D / dt
+    I = I * dt
+    
+    # dimensionalize time
+    t_initial = segment.state.conditions.frames.inertial.time[0,0]
+    t_final   = (vf-v0)/ax # dt
+    t_nondim  = segment.state.numerics.dimensionless.control_points
+    time      = t_nondim * (t_final-t_initial)
+    
+    # Figure out vx
+    vx = v0+time*ax
+    
+    # set the body angle
+    if Tf > T0:
+        body_angle = time*(Tf-T0)/(t_final-t_initial)
+    else:
+        body_angle = T0 - time*(T0-Tf)/(t_final-t_initial)
+    segment.state.conditions.frames.body.inertial_rotations[:,1] = body_angle[:,0]     
+       
+    # pack  
+    t_initial                                       = conditions.frames.inertial.time[0,0]
+    numerics.time.control_points                    = x
+    numerics.time.differentiate                     = D
+    numerics.time.integrate                         = I
+    conditions.frames.inertial.time[:,0]            = time[:,0]
+    conditions.frames.inertial.velocity_vector[:,0] = vx[:,0]
+
+    return
+
+## @ingroup Methods-Missions-Segments-Climb
+def objective(segment):
```

Comment:
Are these methods different enough from these methods?

https://github.com/suavecode/SUAVE/blob/feature-vtol/trunk/SUAVE/Methods/Missions/Segments/Climb/Optimized.py

Reply:
I'm actually going to remove this script.